roles/etcd/tasks/etcdctl.yml

```diff
@@ -4,9 +4,9 @@
   when: not openshift.common.is_atomic | bool
 
 - name: Configure etcd profile.d alises
-  copy:
+  template:
```

Comment:
I have moved the file into template directory. But right, it should be renamed to etcdctl.sh.j2